deploy/helm/sumologic/values.yaml

```diff
@@ -234,6 +234,9 @@ fluentd:
   ## Add custom annotations to all fluentd sts pods(logs, metrics, events)
   podAnnotations: {}
 
+  ## Add custom labels to all fluentd statefulset PVC (logs, metrics, events)
+  pvcLabels: {}
```

Comment:
It would be good to add `pvcLabels` to table available on this page: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/README.md